frontend/src/components/TrainPanel.js

```diff
@@ -145,11 +144,12 @@ const TrainPanel = ({
 
   const handleDnnKwargsChange = (e) => {
     var param = e.target.name;
+    let value;
```

Comment:
Here it is really needed because var declared in "else branch" had already been declared in "if branch". It's strange behavior (hoisting) of var.
Better use let and decalae it at the beginning of the function